nearcore/res/mainnet_genesis.json

```diff
@@ -0,0 +1,823 @@
+{
```

Comment:
`utils/mainnet-res/res/mainnet_genesis.json` is what you’re looking for.  Available through `near_mainnet_res::mainnet_genesis` function.